examples/life-cycle-hooks.yaml

```diff
@@ -0,0 +1,96 @@
+    apiVersion: argoproj.io/v1alpha1
+    kind: Workflow
+    metadata:
+      generateName: lifecycle-hook-
+    spec:
+      entrypoint: diamond
+      hooks:
+        exit:
+          template: http
+        running:
+          expression: "{{workflow.status}} == Running"
```

Comment:
I strongly believe we should be avoiding `{{` if we can. Instead, could this be `workflow.status == 'Running'` instead.